docs/index.rst

```diff
@@ -29,20 +29,333 @@ Asset filters are added in the same way as any other filters.
 
   - Enable the plugin and click *Done* to activate it
 
-Asset Rules
------------
+Rules
+-----
 
-The asset rules are an array of JSON requires **rules** as an array of objects which define the asset name to which the rule is applied and an action. Rules on an asset will be executed in the exact order defined in the JSON array. Actions can be one of
+The asset rules are an array of JSON requires **rules** as an array of objects which define the asset name to which the rule is applied and an action. Rules on an asset will be executed in the exact order defined in the JSON array.
 
-  - **include**: The asset should be forwarded to the output of the filter
+There are a number of different rules that may be applied to the readings, each rule will perform a specific action on the reading and will be applied based upon a match with the asset name within the reading. The rules are configured using a JSON object to define the operation of the rule, each object contains the asset match criteria and the action name. In addition there are a set of parameters that are passed to the rule.
```

Comment:
- Start a new sentence at "Each rule will perform a specific action on the reading..."
- Start a new sentence at "Each object contains the asset match criteria and the action name."